fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMaterializedViewRefreshProcessor.java

```diff
@@ -392,7 +392,8 @@ private InsertStmt generateInsertStmt(Set<String> materializedViewPartitions,
                 .setTimestamp(System.currentTimeMillis())
                 .setClientIp(mvContext.getRemoteIp())
                 .setUser(ctx.getQualifiedUser())
-                .setDb(ctx.getDatabase());
+                .setDb(ctx.getDatabase())
+                .setCatalog(ctx.getCurrentCatalog());
```

Comment:
Internal catalog is not need in the event.

Reply:
done